csx parsing problems

I found a few problems with csx parsing:

1. #load directive with next expression start with string
```csharp
#nullable enable
#load "ScriptHelper.csx"

"Hello world".Dump();
```
Produce error:
`Could not find file '/Users/mark/Code/CSharpLab/ScriptHelper.csx"'.`

Although it works:
```csharp
#nullable enable
#load "ScriptHelper.csx"

;
"Hello world".Dump();
```

2.  using statement without parenthesis or a block on top level
```csharp
using var connection = new SqliteConnection($"Data Source={dbPath}");
```
Produce errors:
`error CS0103: The name 'connection' does not exist in the current context`
`error CS1002: ; expected`

Although it works:
```csharp
void Func()
{
  using var connection = new SqliteConnection($"Data Source={dbPath}");
}
```
3.  [Reference to Microsoft.NET.Sdk.Web not work · Issue #716 · dotnet-script/dotnet-script](https://github.com/dotnet-script/dotnet-script/issues/716)

> > I try to transform my LINQPad queries to dotnet-script. In LINQPad 1, 2 work without problems.
> 
> @i-mark LINQPad doesn't use [C# scripting](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Scripting) like this tool so it's not bound to the same restrictions. LINQPad turns your query/script into a whole C# program, somewhat like what I emulate in [LINQPadless](https://github.com/atifaziz/LinqPadless).

Ok, thx. 

I run it with c# scripting and this code throw (1,11): error CS1002

```csharp
var options = ScriptOptions.Default
		.AddReferences(typeof(Console).Assembly)
		.AddImports("System")
		.AddImports("System.IO");

//var statement = "using (var ms = new System.IO.MemoryStream()) {};";
var statement = "using var ms = new System.IO.MemoryStream();";
await CSharpScript.RunAsync(statement);
```

--- a/src/Dotnet.Script.DependencyModel/ProjectSystem/ScriptFilesResolver.cs
+++ b/src/Dotnet.Script.DependencyModel/ProjectSystem/ScriptFilesResolver.cs
@@ -61,7 +61,7 @@ namespace Dotnet.Script.DependencyModel.ProjectSystem
 
         private static string[] GetLoadDirectives(string content)
         {
-            var matches = Regex.Matches(content, @"^\s*#load\s*""\s*(.+)\s*""", RegexOptions.Multiline);
+            var matches = Regex.Matches(content, @"^\s*#load\s*""\s*(.+?)\s*""", RegexOptions.Multiline);
             List<string> result = new List<string>();
             foreach (var match in matches.Cast<Match>())
             {

--- a/src/Dotnet.Script.Tests/ScriptFilesResolverTests.cs
+++ b/src/Dotnet.Script.Tests/ScriptFilesResolverTests.cs
@@ -21,11 +21,15 @@ namespace Dotnet.Script.Tests
             Assert.Contains(files, f => f.Contains("Foo.csx"));
             Assert.Contains(files, f => !f.Contains("Bar.csx"));
         }
-        [Fact]
-        public void ShouldResolveLoadedScriptInRootFolder()
+
+        [Theory]
+        [InlineData("#load \"Bar.csx\"")]
+        // See: https://github.com/dotnet-script/dotnet-script/issues/720 (1)
+        [InlineData("#load \"Bar.csx\"\n\n\"Hello world\".Dump();")]
+        public void ShouldResolveLoadedScriptInRootFolder(string rootScriptContent)
         {
             using var rootFolder = new DisposableFolder();
-            var rootScript = WriteScript("#load \"Bar.csx\"", rootFolder.Path, "Foo.csx");
+            var rootScript = WriteScript(rootScriptContent, rootFolder.Path, "Foo.csx");
             WriteScript(string.Empty, rootFolder.Path, "Bar.csx");
             var scriptFilesResolver = new ScriptFilesResolver();
 

